<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>344</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>13</number>
     </property>
     <item row="3" column="0">
      <widget class="QComboBox" name="class_CB"/>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="engraving_CB1"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>직업</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QComboBox" name="engraving_CB3"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="artifact_CB"/>
     </item>
     <item row="7" column="3">
      <widget class="QComboBox" name="engraving_CB4"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>스탯</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QComboBox" name="engraving_CB6"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>각인</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QComboBox" name="engraving_CB2">
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>유물 세트</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QComboBox" name="engraving_CB5"/>
     </item>
     <item row="6" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>특화</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stat_SB3"/>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>평균 장비 강화</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stat_SB1">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>치명</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stat_SB2"/>
       </item>
      </layout>
     </item>
     <item row="7" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>신속</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="stat_SB4"/>
       </item>
      </layout>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="remain_stat_L">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
